solidity/test/rewards/TestECDSARewards.js

```diff
@@ -214,6 +214,33 @@ describe("ECDSARewards", () => {
       // 297,000 * 2 = 594,000
       await assertKeepBalanceOfBeneficiaries(expectedBeneficiaryBalance.muln(2))
     })
+
+    it("should correctly receive rewards from multiple keeps", async () => {
+      for (let i = 0; i < 10; i++) {
+        await keepFactory.stubOpenKeep(owner, operators, firstIntervalStart)
+      }
+
+      await timeJumpToEndOfIntervalIfApplicable(0)
+
+      const keepAddress = await keepFactory.getKeepAtIndex(0)
+      const keepAddress1 = await keepFactory.getKeepAtIndex(1)
+      const keepAddresses = [keepAddress, keepAddress1]
+      const keep = await BondedECDSAKeepStub.at(keepAddress)
```

Comment:
Nitpick, not worth blocking on it but given that we have `keep1` I would name this one `keep0`.